docs/1. Introduction/1. About Flamingo Commerce.md

```diff
@@ -0,0 +1,84 @@
+# Flamingo Commerce Graphql API
+
+Most of the modules provide GraphQL schemas and the corresponding resolvers and modifiers.
```

Comment:
Warum wird hier jede Zeile als geändert markiert?
Das gibts doch schon im master?

Btw - sehe gerade das der content in den beiden MArkdowns vertauscht ist

Reply:
Genau, der rebase hat die commits mit reingeholt. Die travis pipe ist wegen einer nicht mehr existierenden dependency ausgestiegen und ich wollte die changes von master nicht manuel machen :)